vault/logical_system_paths.go

```diff
@@ -2052,3 +2052,29 @@ func (b *SystemBackend) mountPaths() []*framework.Path {
 		},
 	}
 }
+
+func (b *SystemBackend) lockerUserPaths() []*framework.Path {
```

Comment:
Did you mean this to be `lockedUserPaths()`?

Reply:
Yes, Thanks :) I will change this